pkg/segment/writer/segstore.go

```diff
@@ -1548,14 +1548,15 @@ func (ss *SegStore) FlushSegStats() error {
 /*
 Encoding Schema for SegStats Single Column Data
 [Version 1B] [isNumeric 1B] [Count 8B] [HLL_Size 4B] [HLL_Data xB]
-[N_type 1B] [Min 8B] [N_type 1B] [Max 8B] [N_type 1B] [Sum 8B]
+Numeric [DType 1B] [Min 8B] [DType 1B] [Max 8B] [NType 1B] [Sum 8B] [NumericCount 8B]
+NonNumeric [DType 1B] [Min_Size 2B] [Min_Data xB] [Max_Size 2B] [Max_Data xB]
```

Comment:
Put a OR here. It will be either numeric or NonNumeric and not both

Reply:
addressed in the latest commit.